packages/cli/package.json

```diff
@@ -2,9 +2,16 @@
   "name": "@pipcook/pipcook-cli",
   "version": "0.5.29",
   "description": "",
-  "main": "index.js",
+  "main": "bin/index",
+  "types": "bin/index",
+  "files": [
+    "dist"
+  ],
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "npm run clean && npm run compile",
```

Comment:
Is the clean required?

Reply:
clean all files building generated before compile again is better IMO :(